Lib/fontbakery/fonts_profile.py

```diff
@@ -26,6 +28,15 @@ def setup_context(files):
             subfiles = glob.glob(pattern)
         for file in subfiles:
             accepted = False
+            # Special case for .ttc files, which add multiple testables
+            # to the context.
+            if file.endswith(".ttc"):
```

Comment:
`.otc` is also sometimes used for font collections.

Reply:
Since this is a merged PR, it would be better to open an issue to propose additional changes. (or even to send a new PR, perhaps)

Reply:
It is just an observation, I don’t have any use for font collections myself.